src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -817,6 +822,615 @@ public void should_alert_if_generated_job_changed() {
         assertThat(driver, hasContent(alert));
     }
 
+    /**
+     * Test if a new ListView is created via Job DSL script TextBox.
+     * The created View shall contain a description.
+     */
+    @Test
+    public void is_ListView_created() {
+        String descriptionText = "This is the description of testView";
+        String jobDslScript = "listView('"+ LIST_VIEW_NAME +"') {\n" +
+                "    description('"+descriptionText+"')\n" +
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        WebElement description = find(By.xpath("//div[@id='description']/div"));
+        Assert.assertThat(description.getText(), containsString(descriptionText));
+    }
+
+    /**
+     * Test if all native columns of a ListView are created and shown correctly.
+     */
+    @Test
+    public void are_columns_created() {
+        Job job1 = createJobAndBuild();
+        String jobDslScript = "listView('"+ LIST_VIEW_NAME +"') {\n" +
+                "  columns {\n" +
+                "    status()\n" +
+                "    weather()\n" +
+                "    name()\n" +
+                "    lastSuccess()\n" +
+                "    lastFailure()\n" +
+                "    lastDuration()\n" +
+                "    buildButton()\n" +
+                "  }\n" +
+                "  jobs{\n" +
+                "    name('"+job1.name+"')\n" +
+                "  }\n" +
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        Assert.assertThat(driver, hasElement(By.xpath("//th[contains(@tooltip, 'Status of the last build')]")));
```

Comment:
The XPATH should be part of the view. (Or is it possible to work with the existing getColumn method?)

Or even better:
```
assertThat(view, hasColumnHeader("Name"))
```